dotcom-rendering/cdk/lib/renderingStack.ts

```diff
@@ -118,6 +120,70 @@ export class RenderingCDKStack extends CDKStack {
 			ttl: Duration.hours(1),
 		});
 
+		const latencyMetric = new Metric({
+			dimensionsMap: {
+				LoadBalancer: ec2app.loadBalancer.loadBalancerFullName,
+				TargetGroup: ec2app.targetGroup.targetGroupFullName,
+			},
+			metricName: 'TargetResponseTime',
+			namespace: 'AWS/ApplicationELB',
+			period: Duration.seconds(30),
+			statistic: 'Average',
+		});
+
+		/** Scaling policies ASCII diagram
+		 *
+		 * Metric value (latency in seconds)
+		 *  0              0.2                  infinity
+		 * ----------------------------------------------
+		 *        - 1       |         + 100%           |
+		 * ----------------------------------------------
+		 * Instance change
+		 *
+		 * -
+		 * When scaling up, we use percentage change (+100% each interval)
+		 * When scaling down, we use absolute change (-1 each interval)
+		 */
+		new StepScalingPolicy(this, 'LatencyScaleUpPolicy', {
+			autoScalingGroup: ec2app.autoScalingGroup,
+			metric: latencyMetric,
+			scalingSteps: [
+				{
+					// No scaling up effect between 0 and 0.2s latency
+					change: 0,
+					lower: 0,
+					upper: 0.2,
+				},
+				{
+					// When latency is higher than 0.2s we scale up by 100%
+					change: 100,
```

Comment:
This is very high no?
Currently out latency is around 0.2, I'm not sure we should double our capacity. Could we add 2 instances instead? That would be 10% of the current cluster size.

Reply:
There is a risk in slowly ramping up (in combination with the cooldown period) that could result in a delayed response to a large spike in requests.

We could have more steps though, so the higher response time the more we scale out?